config/locales/hi-IN.yml

```
@@ -58,7 +58,7 @@ hi-IN:
     - :month
     - :year
   datetime:
-    distance_in_words:
+    distance_of_time_in_words:
```

Hi @barcha47, this line should be `time_ago_in_words:`. Feel free to accept the suggestion.
```suggestion
    time_ago_in_words:
```

---

Thanks for correcting me. Hope will learn more.